index.html

```diff
@@ -1,48 +1,285 @@
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>portfolio</title>
+  </head>
+  <body>
+    <header class="nav">
+      <a href="#">sonu</a>
+      <i class="fa-solid fa-bars"></i>
     </header>
-    <section id="home">
-        
-        <div id="intro">
-        <h1>
-            Hey There. I'm Sonu
-        </h1>
-        <h2>
-            I’am a Software Developer
-        </h2>
+    <section class="main">
+      <h1>Hey There. I`m sonu</h1>
+      <h4>i am a software developer</h4>
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <ul>
+        <li>
+          <a href="#"><i class="fa-brands fa-github"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-solid fa-m"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-solid fa-m"></i></a>
+        </li>
+      </ul>
+    </section>
+
+
+    <section class="work-section">
+      <h1>My Recent Works</h1>
+      <div class="img-container">
+        <img src="./images/image1.png" alt="" />
+        <h2>Multi-Posts Stories</h2>
         <p>
-            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-don’t hestiate to contact me.
-        </p>
-        </div>
-        <ul id="social-media">
-            <li><a href="#"><img src="images/github.svg" id="github"></a></li>
-            <li><a href="#"><img src="images/linkedin.svg" alt=""></a></li>
-            <li><a href="#"><img src="images/Hand.svg" alt=""></a></li>
-            <li><a href="#"><img src="images/twitter.svg" alt=""></a></li>
-            <li><a href="#"><img src="images/M.svg" alt=""></a></li>
-        </ul>
-        
-      </section>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul>
+          <li>
+            <a href="#">css</a>
+          </li>
+          <li>
+            <a href="#">html</a>
+          </li>
+          <li>
+            <a href="#">Bootstraps</a>
+          </li>
+          <li>
+            <a href="#">Ruby</a>
+          </li>
+        </ul>
+        <button>see project</button>

```

Comment:
- [OPTIONAL] This button has a margin that I think can be removed so that it is aligned as in the Figma design. I marked this as optional because it was not asked by the previous reviewer but it should have been implemented before 😅